package.json

```diff
@@ -9,13 +9,13 @@
   "scripts": {
     "publish": "lerna publish",
     "postinstall": "npm run clean && lerna bootstrap --no-ci && npm run style",
-    "clean": "node ./scripts/clean.js",
+    "clean": "npm run clean:node_modules && npm run clean:dist",
+    "clean:dist": "node ./scripts/clean.js",
```

Comment:
Peut etre utiliser un lerna exec couplé avec rimraf (le plus performant sur windows et utiliser par lerna clean)

Reply:
Not sure I get it... 

Rimraf being used in both `lerna clean` and `clean.js` what would you like?

I am not sure we should rewrite `lerna clean`. If any bug, let's fix it in Lerna's repo.

As for `clean.js`, if any bug / improvement, let's fix it in another PR?